tasks/get_current_archive_list/test/unit_tests/test_get_current_archive_list.py

```diff
@@ -994,15 +1001,15 @@ def test_handler_happy_path(
         with patch.dict(
             os.environ,
             {
-                get_current_archive_list.OS_ENVIRON_S3_CREDENTIALS_SECRET_ARN_KEY: secret_arn,
+                get_current_archive_list.OS_ENVIRON_S3_CREDENTIALS_SECRET_ARN_KEY: db_connect_info_secret_arn,
                 get_current_archive_list.OS_ENVIRON_INTERNAL_REPORT_QUEUE_URL_KEY: report_queue_url,
             },
         ):
             result = get_current_archive_list.handler(event, mock_context)
 
         mock_LOGGER.setMetadata.assert_called_once_with(event, mock_context)
-        mock_get_s3_credentials_from_secrets_manager.assert_called_once_with(secret_arn)
-        mock_get_configuration.assert_called_once_with()
+        mock_get_s3_credentials_from_secrets_manager.assert_called_once_with(db_connect_info_secret_arn)
+        mock_get_configuration.assert_called_once_with(os.environ["DB_CONNECT_INFO_SECRET_ARN"])
```

Comment:
Check indent spacing on lines 1011 and 1012. It could be the gitdiff makes it look incorrect but it appears that they are not indented the proper number of spaces.

Reply:
indentation looks good to me.